SOPT-iOS/Projects/Modules/Network/Sources/Foundation/BaseService.swift

```diff
@@ -131,6 +151,42 @@ extension BaseService {
         }.eraseToAnyPublisher()
     }
     
+    func opRequestObjectInCombine<T: Decodable>(_ target: API) -> AnyPublisher<T, Error> {
+        return Future { promise in
+            self.provider.request(target) { response in
+                switch response {
+                case .success(let value):
+                    do {
+                        let decoder = JSONDecoder()
+                        let body = try decoder.decode(T.self, from: value.data)
+                        promise(.success(body))
+                    } catch let error {
+                        promise(.failure(error))
+                    }
+                case .failure(let error):
+                    // NOTE: 에러 메세지 받아서 보여주기 위함
+                    do {
+                        let decoder = JSONDecoder()
+                        let errorData = try decoder.decode(BaseEntity<Data>.self, from: error.response?.data ?? Data())
+                        throw OPAPIError.attendanceError(errorData)
+                    } catch let error {
+                        promise(.failure(error))
+                    }
```

Comment:
질문있습니당 요기 catch 하는 부분에서 에러 thorw하는데 예외가 반환이 안되고 다음 라인으로 넘어가는데 return 추가하고 함수 나가도록 해야하는거 아닌가요오...??
```suggestion
                    } catch let error {
                         promise(.failure(error))
                         return
                     }
```

Reply:
아래 케이스도 확인해야할 것 같아서
retryError가 아니라면 error 방출하는 코드만 제거해줄게요 !